### DISCUSSION TITLE
Where is the DaSiamRPN and SiamRPN++ code?

### DISCUSSION THREAD

**Discussion by StrugglingForBetter** [STVIR/pysot#372] [General]
Where is the DaSiamRPN and SiamRPN++ code? I didn't found that. Can anybody share the code with me?

**Comment by ZhiyuanChen** [comment]
What exactly do you mean by DaSiamRPN and SiamRPN++ code?

**Comment by StrugglingForBetter** [comment]
I mean it's really hard to find DaSiamRPN for people who are not famililar with the algorithm because all folders are named siamrpn.

**Comment by ZhiyuanChen** [comment]
> I mean it's really hard to find DaSiamRPN for people who are not famililar with the algorithm because all folders are named siamrpn.

This is beacuse it is a combination of all algorithms. We might make them seperable.
